src/modules/local_position_estimator/sensors/vision_velocity.cpp

```diff
@@ -0,0 +1,109 @@
+#include "../BlockLocalPositionEstimator.hpp"
+#include <systemlib/mavlink_log.h>
+#include <matrix/math.hpp>
+
+extern orb_advert_t mavlink_log_pub;
+
+static const uint32_t 		REQ_VISION_VEL_INIT_COUNT = 10;
+static const uint32_t 		VISION_VELOCITY_TIMEOUT =    500000;	// 0.5 s
```

Comment:
@ecmnet Have this set to 0.5 sec,  so should be  what you want, maybe we should make it s param, but just worried about mem for smaller boards.